gnucash/report/report-gnome/report-gnome.scm

```diff
@@ -102,7 +103,8 @@
   (gnc-add-scm-extension income-expense-menu)
   (gnc-add-scm-extension asset-liability-menu)
   (gnc-add-scm-extension budget-menu)
-  (gnc-add-scm-extension utility-menu)
+  (if (gnc-prefs-is-extra-enabled)
+    (gnc-add-scm-extension example-menu))
```

Comment:
@christopherlam This line hides the complete "Examples" menu unless users set the --extra command line.